src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -174,19 +187,39 @@ export const ArtworkFilterContextProvider: React.FC<
     initialFilterState
   )
 
+  const [stagedArtworkFilterState, stage] = useReducer(artworkFilterReducer, {})
+
   // TODO: Consolidate this into additional reducer
   const [filterAggregations, setAggregations] = useState(aggregations)
   const [artworkCounts, setCounts] = useState(counts)
+  const [shouldStageFilterChanges, setShouldStageFilterChanges] = useState(
```

Comment:
Echoing a meta comment (which is covered by the TODO above, for a separate PR) is we should eventually merge these individual state calls into a reducer so we don't have all of these different state patterns flowing through the filter, and instead explicitly manage updates via a common interface of typed actions that are dispatched.